libopenage/engine/engine.h

```diff
@@ -72,11 +74,13 @@ class Engine {
 	 * @param root_dir openage root directory.
 	 * @param mods The mods to load.
 	 * @param debug_graphics If true, enable OpenGL debug logging.
+	 * @param window_settings window display setting
 	 */
 	Engine(mode mode,
 	       const util::Path &root_dir,
 	       const std::vector<std::string> &mods,
-	       bool debug_graphics = false);
+	       bool debug_graphics = false,
+	       const renderer::window_settings &window_settings = {});
```

Comment:
`debug_graphics` can now be removed, since it is already part of `window_settings`.